src/openrct2-ui/interface/Widget.cpp

```diff
@@ -569,6 +569,8 @@ namespace OpenRCT2::Ui
         topLeft.x += width / 2;
         if (Config::Get().interface.WindowButtonsOnTheLeft)
             topLeft.x += kCloseButtonSize;
+        if (Config::Get().interface.EnlargedUi)
+            topLeft.y += 6;
```

Comment:
This looks like a magic number.

Reply:
Yeah, I think you were compensating the caption to adjust for the enlarged title height in this way. I've changed it to be expressed in terms of `kTitleHeightDouble`.  

Perhaps the widget should just be vertically centred at all times, though, like we do with button captions? It's complicated by the outline, but that'd just be an added `-1`.